<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoadFileDialogue</class>
 <widget class="QDialog" name="LoadFileDialogue">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>171</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>171</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Load Wallet</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="1,10">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="hLayout_FileSelection">
     <item>
      <widget class="QLineEdit" name="lineEdit_FilePath">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="placeholderText">
        <string>Wallet File Path</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_Browse">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="vLayout_Encryption">
     <item>
      <widget class="QComboBox" name="comboBox_AESMode">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>No Encryption</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>AES 128</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>AES 192</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>AES 256</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_KeyMode">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>SHA-256 Digest</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Direct Input</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_EncryptionKey">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>Encryption Key</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_LoadFile">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>Load File</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
